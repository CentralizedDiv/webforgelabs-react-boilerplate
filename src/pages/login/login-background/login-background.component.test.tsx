import React from 'react';
import LoginBackground from './login-background.component';

describe('LoginBackground', () => {
  const component = <LoginBackground/>;

  it('renders properly', () => {
    expect(component).toMatchSnapshot();
  });

});