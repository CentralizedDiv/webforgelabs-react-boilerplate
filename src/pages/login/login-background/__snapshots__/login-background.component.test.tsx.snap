// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginBackground renders properly 1`] = `<LoginBackground />`;
